.github/workflows/ci_python.yml

```diff
@@ -61,13 +61,13 @@ jobs:
           poetry install --only=main --only=lint
       - name: black
         run: |
-          poetry run black --diff pysrc pytests docs/source
```

Comment:
Are we sure we don't want to lint the python code in the docs? This seems useful for ensuring that all of the example code we include looks good, etc.

Reply:
this is still running on docs source, but I stopped `flake8` due to it complaining about several issues in the `_scripts` folder that are related to how the code was written.  

I'll re-enable `flake8` on `docs` and just mark the `_scripts` as being excluded.